modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Comments/Comment.cs

```diff
@@ -23,6 +23,8 @@ public class Comment : AggregateRoot<Guid>, IHasCreationTime, IMustHaveCreator,
 
     public virtual DateTime CreationTime { get; set; }
 
+    public virtual string CommentUrl { get; set; } 
```

Comment:
It should be `Url`. (*CommentUrl* is meaningless, because we define the property in the Comment entity)